src/PrestaShopBundle/Controller/Admin/ModuleController.php

```diff
@@ -134,9 +134,14 @@ public function manageAction()
         // Retrieve current shop
         $shopID = $shopService->getContextShopID();
         $shops = $shopService->getShops();
-        $shop = $shops[$shopID];
-        $currentTheme = $themeRepository->getInstanceByName($shop['theme_name']);
-        $modulesTheme = $currentTheme->getModulesToEnable();
+
+        if (!empty($shopID) && is_array($shops) && array_key_exists($shopID, $shops)) {
+            $shop = $shops[$shopID];
+            $currentTheme = $themeRepository->getInstanceByName($shop['theme_name']);
+            $modulesTheme = $currentTheme->getModulesToEnable();
+        } else {
+            $modulesTheme = false;
```

Comment:
Why don't you just initialize this variable as an empty array ? You won't have to change the test later l.157